activerecord/lib/active_record/fixtures.rb

```diff
@@ -456,6 +456,31 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError # :nodoc:
   # In the above example, 'base' will be ignored when creating fixtures.
   # This can be used for common attributes inheriting.
   #
+  # == Composite Primary Key Fixtures
+  #
+  # If a model has a composite primary key, both composite key columns should
+  # be specified in its fixtures:
+  #
+  #   # app/models/order.rb
+  #   class Order < ApplicationRecord
+  #     self.primary_key = [:shop_id, :number]
+  #   end
+  #
+  #   # orders.yml
+  #   shop_id: <%= ActiveRecord::FixtureSet.identify(:book_store) %>
```

Comment:
foreign keys of a non CPK model can actually use the "association name resolution"
```suggestion
  #   shop: book_store
```

What we would like to highlight in the guides is the use of `composite_identify` like
https://github.com/rails/rails/blob/4c5747377c503d50b512161ff46ed6509cf04493/activerecord/test/fixtures/cpk_order_agreements.yml#L11

Reply:
Oh that's right. Linking them to other models is less obvious. I'll add that! If we want to use association naming for the shop relation we can do that too, but the shard key sometimes doesn't have an association backing it. We can add it to the model example though.